### PR TITLE
MOSIP-34248: Added certificates expiry check in getOriginalPartnerCer…

### PR DESCRIPTION
**Pull request by SwethaKrish4**
…tificates()

### CODE REVIEW

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/impl/PartnerServiceImpl.java [hunk 1]
```diff
@@ -805,8 +807,26 @@ public PartnerCertDownloadResponeDto getPartnerCertificate(PartnerCertDownloadRe
 
 	@Override
 	public OriginalCertDownloadResponseDto getOriginalPartnerCertificate(PartnerCertDownloadRequestDto certDownloadRequestDto)
-			throws JsonParseException, JsonMappingException, JsonProcessingException, IOException {
-		return getCertificateFromKeyMgr(certDownloadRequestDto, "pmp.partner.original.certificate.get.rest.uri", OriginalCertDownloadResponseDto.class);
+			throws JsonParseException, JsonMappingException, JsonProcessingException, IOException, CertificateException {
+		OriginalCertDownloadResponseDto responseDto = getCertificateFromKeyMgr(certDownloadRequestDto, "pmp.partner.original.certificate.get.rest.uri", OriginalCertDownloadResponseDto.class);
+		responseDto.setIsMosipSignedCertificateExpired(false);
+		responseDto.setIsCaSignedCertificateExpired(false);
+		LocalDateTime currentDateTime = LocalDateTime.now(ZoneId.systemDefault());
```
Comment (mayuradesh) [inline partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/impl/PartnerServiceImpl.java:814]: ZoneId should be UTC

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java [hunk 2]
```diff
@@ -356,7 +357,7 @@ public ResponseWrapper<PartnerCertDownloadResponeDto> getPartnerCertificate(
 	@RequestMapping(value = "/{partnerId}/originalPartnerCertificate", method = RequestMethod.GET)
 	@Operation(summary = "Service to get original partner certificate", description = "Service to get original partner certificate")
 	public ResponseWrapper<OriginalCertDownloadResponseDto> getOriginalPartnerCertificate(
-			@ApiParam("To download original partner certificate.")  @PathVariable("partnerId") @NotNull String partnerId) throws JsonParseException, JsonMappingException, JsonProcessingException, IOException {
+			@ApiParam("To download original partner certificate.")  @PathVariable("partnerId") @NotNull String partnerId) throws JsonParseException, JsonMappingException, JsonProcessingException, IOException, CertificateException {
```
Comment (mayuradesh) [inline partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java:360]: Have u tested the CertificateException case?
What is the API response in this case?

**Review by mayuradesh — CHANGES_REQUESTED** [review]
added comments

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java (hunk 2 above)
Comment (SwethaKrish4) [reply partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java:360]: Yes, Getting 200 response from API

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/impl/PartnerServiceImpl.java (hunk 1 above)
Comment (SwethaKrish4) [reply partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/impl/PartnerServiceImpl.java]: updated the ZoneId in UTC

File: partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java (hunk 2 above)
Comment (mayuradesh) [reply partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java:360]: Add proper error code and error mesage

**Review by mayuradesh — APPROVED** [review]

**Pull request merged**